contentcuration/contentcuration/static/js/edit_channel/channel_set/vuex/store.js

```diff
@@ -0,0 +1,36 @@
+import Vue from 'vue';
+var Vuex = require('vuex');
+var mutations = require('./mutations');
+var actions = require('./actions');
+var getters = require('./getters');
+var { PageTypes } = require('../constants');
+Vue.use(Vuex);
+
+var store = new Vuex.Store({
```

Comment:
In general, it would be better to have a single store, and then register and unregister modules against that central store. This will work for now, but as we build up more complex state, this will be necessary, as if you have distinct stores, components will only _know_ about the store they have been initialized with.

Reply:
Sounds good! I'll add it to the technical debt section on this PR